doc/source/Utilities.rst

```diff
@@ -9,16 +9,14 @@ Miscellaneous utilities
 show_duplicate_packages.py
 ------------------------------
 
-The utility located at util/show_duplicate_packages.py parses the output of ``spack concretize`` and detects duplicates. Usage is as follows:
+The utility located at util/show_duplicate_packages.py parses ``spack.lock`` and detects duplicates. Usage is as follows:
 
 .. code-block:: console
 
-   spack concretize | ${SPACK_STACK_DIR}/util/show_duplicate_packages.py
-   # - OR -
-   spack concretize |& tee log.concretize
-   ${SPACK_STACK_DIR}/util/show_duplicate_packages.py log.concretize
+   # In $SPACK_ENV, after concretization:
```

Comment:
I think this is misleading - you don't need to be in the directory ${SPACK_ENV}, but you need to have an environment activated so that ${SPACK_ENV} is defined?

Reply:
Good catch, thanks